src/libcsg/exclusionlist.cc

```diff
@@ -49,7 +49,7 @@ void ExclusionList::CreateExclusions(Topology *top) {
 
 bool ExclusionList::IsExcluded(Bead *bead1, Bead *bead2) {
   exclusion_t *excl;
-  if (bead1->getMolecule() != bead2->getMolecule()) {
+  if (bead1->getMoleculeId() != bead2->getMoleculeId()) {
```

Comment:
this is actually interesting because these two comparisons are not the same. The old checks if the two pointers point to the same molecule while the new one only compares the ids. Is that a problem?

Reply:
No, exclusions should be based on the id not a pointer, they apply no matter if the bead is a copy or not.

Reply:
okay nice

Reply:
Puh, we merged the same change in https://github.com/votca/csgapps/pull/56

Reply:
I mean you guys have more knowledge, I just saw it :)

Reply:
@junghans how can we have that merge? It should fail?

Reply:
I think Christoph already merged my test branch in votca/votca repo with master, so it is building against commit 3d4c110, I think.